docs/api/en/loaders/TextureLoader.html

```diff
@@ -85,8 +85,8 @@ <h3>[method:Texture load]( [param:String url], [param:Function onLoad], [param:F
 		[page:String url] — the path or URL to the file. This can also be a
 			[link:https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs Data URI].<br />
 		[page:Function onLoad] — Will be called when load completes. The argument will be the loaded [page:Texture texture].<br />
-		[page:Function onProgress] — Will be called while load progresses. The argument will be the XMLHttpRequest instance, which contains .[page:Integer total] and .[page:Integer loaded] bytes. If the server does not set the Content-Length header; .[page:Integer total] will be 0.<br />
-		[page:Function onError] — Will be called when load errors.<br /><br />
+		[page:Function onProgress] (optional) — This callback function is currently not supported.<br />
```

Comment:
The `onLoad()` callback is sometimes optional. Like with `TextureLoader`. Meaning this is valid code:
```js
const texture = new THREE.TextureLoader().load( 'textures/crate.gif' );
```

Reply:
Okay I modified it on all texture loaders. And added the phrase "The method also returns a new texture object which can directly be used for material creation." when missing.